themes/default/content/blog/top-5-things-for-azure-devs-vm/index.md

```diff
@@ -0,0 +1,231 @@
+---
+title: "Top 5 an Azure Developer Needs to Know: Virtual Machines"
+date: 2021-07-29
+meta_desc: "Building and deploying virtual machines for Azure developers"
+meta_image: azure-top-5.png
+authors:
+    - sophia-parafina
+tags:
+    - azure
+    - virtual machines
+---
+
+So you want to be an Azure dev and all-around infrastructure wizard? Let's start with the basics, virtual machines! In the previous article, the common use case for virtual machines is migrating applications from dedicated hardware. You want full control of the machine to install required software for the application or configure storage and networking.
```

Comment:
Should "the previous article" link to the previous article?

Reply:
link added